lib/.gemspec.in

```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
+
+Gem::Specification.new do |s|
+  s.name = "facter"
+  version = "@LIBFACTER_VERSION_MAJOR@.@LIBFACTER_VERSION_MINOR@.@LIBFACTER_VERSION_PATCH@"
+  mdata = version.match(/(\d+\.\d+\.\d+)/)
+  s.version = mdata ? mdata[1] : version
+
+  s.required_rubygems_version = Gem::Requirement.new(">= 0")
+  s.authors = ["Puppet Labs"]
+  s.date = "2012-08-08"
+  s.description = "You can prove anything with facts!"
+  s.email = "info@puppetlabs.com"
+  s.executables = ["facter"]
+  s.files = ["bin/facter"]
```

Comment:
That's a really good question and to be honest I'm not sure what the contract really is for this field (or most of the others tbh). The docs for this field are at http://guides.rubygems.org/specification-reference/#files but again the exact semantics frankly aren't that clear to me.

Thoughts on what fields we actually *need* to provide here? I'd really rather pare this down to what is actually needed.

Btw, if we agree on a set, we may want to apply these lessons to the actual gem projects (puppet and hiera) as well. I already applied some fixup to the puppet gemspec in https://github.com/puppetlabs/puppet/pull/4223, but I suspect there's more cargo cult to be purged, both in a real gem install and in a puppet-agent install.